2.6.md

```diff
@@ -3076,10 +3076,11 @@ This object allows sellers to signal interest group auction support for an Imp.
   </tr>
   <tr>
     <td><code>ae <b>PROVISIONAL</b></code></td>
-    <td>integer;<br>default 1</td>
+    <td>integer;<br>default 0</td>
     <td>Interest Group auction environment support for this impression: <br>
-1 = on-device orchestrated auction<br>
-2 = server-orchestrated auction<br>
+0 = server-side auction without interest groups<br>
```

Comment:
WIth the _InterestGroupAuctionSupport_ object there is no need for a "0" signal. When the _Imp_ presents this object there is some IG support. No object, no support.

Reply:
Without this, you remove the ability for an exchange or publisher to say there is definitely not an interest group auction coming. Null suggests 'there might be one, who knows? '

Reply:
>Null suggests 'there might be one, who knows? '

By not sending an `InterestGroupAuctionSupport` object the exchange or publisher is signaling that there is no support. Should we add wording to that effect to make this explicit?

Reply:
How would the SSP signal it doesn't know if there will be an interest group auction; which might be typical? This is currently how ae=1 is used by the prebid SSPs.

I am not sure I follow the objection to 0, in which one is explicitly signaling there will not be one. That is different than a missing object.

Reply:
Agreed with @patmmccann here!

Reply:
> Do you envision this to effectively signal there definitely not a runAdAuction call associated with that slot?

This seems useful to signal, no?